eng/common/performance/perfhelixpublish.proj

```diff
@@ -64,21 +64,54 @@
     <WorkItemCommand>$(WorkItemCommand) $(CliArguments)</WorkItemCommand>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(HelixSourcePrefix)'' == 'pr'">
+    <WorkItemTimeout>0:15</WorkItemTimeout>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(HelixSourcePrefix)'' == 'official'">
```

Comment:
If for some reason `HelixSourcePrefix` is not pr nor official it will use the default HelixSDK timeout, maybe we want to be intentional about it. Something like:

```xml
<PropertyGroup>
   <WorkItemTimeout>0:45</WorkItemTimeout>
  <WorkItemTimeout  Condition="'$(HelixSourcePrefix)' != 'official'">0:15</WorkItemTimeout>
</PropertyGroup>
```

Reply:
SGTM